css/_filmstrip.scss

```diff
@@ -60,6 +60,7 @@
         font-size: 0pt; /*!!!Removes the gap between the local video container and the remote videos.*/
 
         &.hidden {
+            display: block; // TOFIX: overwrites aui styles
```

Comment:
This line is fixing bug introduced after I move `bundle.js` to body. As we link AUI css from `bundle.js` it has more specificity than our CSS file (`all.css`). There are more bugs introduced by this change. I think we can fix it by linking to AUI styles from `index.html` instead of `bundle.js`